rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf

```diff
@@ -98,6 +98,22 @@ SecRule REQUEST_FILENAME "@rx ^/wp\-json/wp/v[0-9]+/(?:posts|pages)" \
     ctl:ruleRemoveTargetByTag=CRS;ARGS:content,\
     ctl:ruleRemoveTargetByTag=CRS;ARGS:json.content"
 
+# Gutenberg via rest_route for sites without pretty permalinks
+SecRule REQUEST_FILENAME "@endsWith /index.php" \
+    "id:9002141,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    chain"
+    SecRule &ARGS:rest_route "@eq 1" \
+        "t:none,\
+        nolog,\
+        chain"
+    SecRule ARGS:rest_route "@rx ^/wp/v[0-9]+/(?:posts|pages)" \
```

Comment:
Style: this should be indented 4 spaces deeper.

Reply:
Oopsie.